numpy/_core/tests/test_indexing.py

```diff
@@ -534,8 +534,8 @@ def test_indexing_array_weird_strides(self):
     def test_indexing_array_negative_strides(self):
         # From gh-8264,
         # core dumps if negative strides are used in iteration
-        arro = np.zeros((4, 4))
-        arr = arro[::-1, ::-1]
+        arr0 = np.zeros((4, 4))
+        arr = arr0[::-1, ::-1]
```

Comment:
?

Reply:
see above and specifically commit  8afd1bf9e4aac96eb3e91f62c0b5ed7bd0b2f495